object_store/src/buffered.rs

```diff
@@ -255,7 +259,8 @@ impl BufWriter {
         }
     }
 
-    /// Returns the [`MultipartId`] if multipart upload
+    /// Returns the [`MultipartId`] of the last multipart upload created by this
```

Comment:
I didn't completely follow the code here. 

```suggestion
    /// Returns the [`MultipartId`] of multipart upload created by this
```